src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -879,7 +879,7 @@ private void ValidateUnsafeParameters(BindingDiagnosticBag diagnostics, Immutabl
                 var numParametersToCheck = Math.Min(targetParameterTypes.Length, ParameterCount);
                 for (int i = 0; i < numParametersToCheck; i++)
                 {
-                    if (targetParameterTypes[i].Type.IsUnsafe())
+                    if (targetParameterTypes[i].Type.ContainsPointer())
```

Comment:
Note: this is a breaking change. This is covered by a recent entry in the doc for this umbrella (previously unreported usage of pointer types).